Gem/Code/Source/AutoGen/NetworkRandomComponent.AutoComponent.xml

```diff
@@ -8,6 +8,6 @@
     OverrideInclude="Source/Components/NetworkRandomComponent.h"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 
-    <NetworkProperty Type="uint64_t" Init="0" Name="Seed" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="false" IsPredictable="false" ExposeToEditor="false" ExposeToScript="false" GenerateEventBindings="false" Description="The RNG seed" />
+    <NetworkProperty Type="uint64_t" Init="0" Name="Seed" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="false" IsPredictable="true" ExposeToEditor="false" ExposeToScript="false" GenerateEventBindings="false" Description="The RNG seed" />
```

Comment:
IsPredicable = true. That gives autonomous write access right? What if the player roles the next number and then does something that affects the mp state before the server also got a chance to role the same number? Does it also need to be rewindable?

Reply:
Yup, we're hoping that the two stay in sync via determinism of the rolls and that desyncs are corrected at the level of the affected value rather than against the seed.